webcompat/static/js/lib/bugform.js

```diff
@@ -11,6 +14,21 @@ function BugForm() {
   this.UPLOAD_LIMIT = 1024 * 1024 * 4;
   this.clickedButton = null;
   this.hasImage = null;
+  // image editor props
+  this.editImageButton = $(".js-edit-img-open-btn");
+  this.editImageModal = $("#imageEditor");
+  this.editImageCloseButton = $(".js-edit-img-close-btn");
+  this.editImageHighlightButton = $(".js-highlight-button");
+  this.editImageRedactButton = $(".js-redact-button");
+  this.canvas = $("#mCanvas");
+  this.curEditorStateOn = true;
+  this.ACTIVE_CONTROL_OPTIONS = [
```

Comment:
Using this array throughout the code feels a bit awkward -- I'm not sure what it really buys us (and later on reading `this.ACTIVE_CONTROL_OPTIONS[0]` is harder to understand than something like `this.HIGHLIGHT_CONTROL` (or something)).